jetty-server/src/main/config/etc/jetty-threadpool.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0"?>
+<!DOCTYPE Configure PUBLIC "-//Jetty//Configure//EN" "http://www.eclipse.org/jetty/configure_9_3.dtd">
+
+<Configure id="Server" class="org.eclipse.jetty.server.Server">
```

Comment:
No, this does not work.

We want to be able to create a custom thread pool, so it should be:

```xml
<Configure id="threadPool" class="org.eclipse.jetty.util.thread.QueuedThreadPool">
  <Set name="minThreads" type="int"><Property name="jetty.threadPool.minThreads" default="10"/></Set>
  ...
</Configure>
```